cli/services/StaticGenerator/StaticGenerator.js

```diff
@@ -190,51 +222,58 @@ class StaticGenerator {
                 }
               })
             )
-            .pipe(gulp.dest(this.buildPath + resource.dest))
+            .pipe(gulp.dest(path.join(this.buildPath, resource.dest)))
             .on('end', resolve);
         });
       })
     );
   }
 
+
+  // TODO: Figure out why this code throws a PluginError
```

Comment:
I will look at replacing some of this with the custom gulp logic I have been improving on in the past.